src/any_agent/frameworks/tinyagent.py

```diff
@@ -180,10 +180,11 @@ async def _run_async(self, prompt: str, **kwargs: Any) -> str | BaseModel:
 
         num_of_turns = 0
         max_turns = kwargs.get("max_turns", DEFAULT_MAX_NUM_TURNS)
+        completion_params = self.completion_params.copy()
```

Comment:
I assume we want the `self.completion_params` to be defaults, right?

My assumption is that a deep copy is not necessary because all params will be immutable; just in case.